tests/e2e/env/bin/wc-e2e.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+#
+# Run package scripts
+#
+
+# Script help
+usage() {
+	echo 'usage: npx wc-e2e <script>'
+	echo 'scripts:'
+	echo '         docker:up [initialization-script] - boot docker container'
+	echo '         docker:down - shut down docker container'
+	echo '         docker:ssh - open SSH shell into docker container'
+	echo '         docker:clear-all - remove all docker containers'
+	echo '         test:e2e [test-script] - run e2e test suite or specific test-script'
+	echo '         test:e2e-dev [test-script] - run e2e test(s) in non-headless mode'
+	echo '         test:e2e-debug [test-script] - run e2e test(s) in non-headless debug mode'
+}
+
+# Parameter check
+if [ $# -lt 1 ]; then
+	usage
+	exit 1
+fi
+
+# Store original path
+OLDPATH=$(pwd)
+
+# Use the script symlink to find and change directory to the root of the package
+SCRIPTPATH=$(dirname "$0")
+REALPATH=$(readlink $0)
+cd $SCRIPTPATH/$(dirname "$REALPATH")/..
+
+# Run scripts
+case $1 in
+	'docker:up')
+		./bin/docker-compose.sh up $2
+		;;
+	'docker:down')
+		./bin/docker-compose.sh down
+		;;
+	'docker:ssh')
+		docker exec -it $(node utils/get-app-name.js)_wordpress-www /bin/bash
+		;;
+	'docker:clear-all')
+		docker rmi --force $(docker images -q)
```

Comment:
Doesn't `docker images -q` return the ID of _every_ image on the user's system? The name of this command implies it will clean up the package's environment but if it deletes all of the images on my system that seems inappropriate.

Reply:
`docker images` has a `--filter` argument, but I don't currently see a way to figure out what images we should care about ... 🤔

Reply:
> The name of this command implies it will clean up the package's environment but if it deletes all of the images on my system that seems inappropriate.

This isn't a new command. You can currently run it with `npm explore @woocommerce/e2e-environment -- npm run docker:clear-all` or run it directly from the `env` folder. We could not include it in the `wc-e2e` script.